.krew.yaml

```diff
@@ -61,7 +61,9 @@ spec:
           to: .
         - from: LICENSE
           to: .
-  shortDescription: Manage Accurate features
+  shortDescription: Accurate is a Kubernetes controller for multi-tenancy.
   description: |
     kubectl-accurate is a subcommand of kubectl to manage Accurate features.
-    Read more documentation at: https://cybozu-go.github.io/accurate/kubectl-accurate.html
+    Accurate is a Kubernetes controller for multi-tenancy.
+    It propagates resources between namespaces accurately and allows tenant users to create/delete sub-namespaces.
+    Read more documentation at: https://cybozu-go.github.io/accurate/index.html
```

Comment:
In the pull request to add to moco's krew index, it was pointed out to me, so I made the description more detailed.
https://github.com/kubernetes-sigs/krew-index/pull/1716#discussion_r741192314

Reply:
https://github.com/kubernetes-sigs/krew-index/pull/1721#pullrequestreview-797095195